resources/multisite_drupal_standard.make

```diff
@@ -349,9 +349,6 @@ projects[media][patch][] = https://www.drupal.org/files/issues/media-wysiwyg-ove
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-3650
 projects[media][patch][] = http://www.drupal.org/files/issues/file_entity-remove-contextual-links-2401811-11.patch
 
-projects[media_crop][subdir] = "contrib"
```

Comment:
if ``media_crop`` is removed from the platfrome, the uninstall of this module cannot be done